abnormality_detection/abnormality_detection.py

```diff
@@ -83,3 +82,17 @@ def judge_image(model: Dict[str, Any], image: Image.Image, allowed_regions: np.n
 
     judge_positions(detected_objects, allowed_regions)
     return detected_objects, visualization
+
+
+def plot_object(judged_items: List[Dict[str, Any]], allowed_regions_rgb: Image.Image):
+    draw = ImageDraw.Draw(allowed_regions_rgb)
+    for item in judged_items:
+        x = round(item['x']) - 1
+        y = round(item['y']) - 1
+        if item['inside_allowed_region']:
+            # mark position in blue
+            draw.regular_polygon((x, y, 3), 6, rotation=0, fill='blue', outline=None)
```

Comment:
I think drawing a circle is better
```suggestion
            draw.ellipse((x-3, y-3, x+3, y+3), fill='blue', outline=None)
```